ui/core.js

```diff
@@ -157,13 +157,13 @@ $.extend( $.expr[ ":" ], {
 		},
 
 	focusable: function( element ) {
-		return focusable( element, !isNaN( $.attr( element, "tabindex" ) ) );
+		return focusable( element, $.attr( element, "tabindex" ) != null );
```

Comment:
For our intents, yes. We assume that if the attribute exists, it contains a valid value. If someone were to file a bug saying that they did something like `tabindex="bananas"` and it resulted in our selectors finding incorrect elements, we'd close it the same as if they filed a bug about our widgets not working properly when they used the same `id` on multiple elements :-)